### PR TITLE
Fixes revival stasis sometimes not displaying confirmation popup when alive

### PR DESCRIPTION
**Pull request by ghost**
Also renames the stasis to Regenerative.

Finally, renames Epinephrine Overdose to Adrenaline Overdose and tweaks its effects a bit.

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/fakedeath.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /obj/effect/proc_holder/changeling/fakedeath
-	name = "Revival Stasis"
```
Comment (Anonus) [inline code/game/gamemodes/changeling/powers/fakedeath.dm]: Paprika did this!

**Comment by Iamgoofball** [comment]
can you rename epinephrine overdose to something else while you're at it too

as how it's named implies it actually overdoses you on epi, which is a Bad Thing :tm: now

**Comment by Xhuis** [comment]
sure

**Comment by Cheridan** [comment]
What's wrong with Revival? I think the word is more indicative of the fact it can bring you back from the dead.

**Comment by Xhuis** [comment]
There's nothing wrong with it. Some people dislike it, however, and wanted to go back to Regenerative. If you personally have a problem with it, I can change it to Revival - there's no problem with that.

**Pull request merged**